release/_bom.sh

```diff
@@ -224,58 +224,55 @@ function generate_pom_release_dxp_bom_third_party {
 		-e "w ${pom_file_name}" \
 		"${_RELEASE_TOOL_DIR}/templates/release.dxp.bom.third.party.pom.tpl" > /dev/null
 
-	local property_files=(
-		"${_PROJECTS_DIR}/liferay-portal-ee/lib/development/dependencies.properties"
-		"${_PROJECTS_DIR}/liferay-portal-ee/lib/portal/dependencies.properties"
-	)
+	IFS=$'\n' read -d '' -r -a development_properties < "${_PROJECTS_DIR}/liferay-portal-ee/lib/development/dependencies.properties"
 
-	find "${_BUNDLES_DIR}/tomcat/webapps/ROOT/WEB-INF/shielded-container-lib" -name "*.jar" -print0 | \
-		while IFS= read -d '' -r jar_file
-		do
-			if (unzip -l "${jar_file}" | grep -q "pom.xml$")
-			then
-				local artifact_name="${jar_file##*/}"
-
-				artifact_name="${artifact_name/%\.jar}"
-
-				if [[ "${artifact_name}" == com.liferay.* ]]
-				then
-					continue
-				fi
-
-				artifact_name="${artifact_name##*.}"
-
-				local artifact_properties=$(grep -w "^$artifact_name" "${property_files[@]}")
-
-				if [ -z "${artifact_properties}" ]
-				then
-					continue
-				fi
-
-				if [[ "${artifact_properties}" == *development/dependencies.properties* ]] && [[ "${artifact_properties}" == *portal/dependencies.properties* ]]
-				then
-					artifact_properties=$(echo "${artifact_properties}" | grep "portal/dependencies.properties" | cut -d= -f2)
-				else
-					artifact_properties=$(echo "$artifact_properties" | cut -d= -f2)
-				fi
-
-				echo "$artifact_properties" | \
-					while IFS=: read -r group_id artifact_id version
-					do
-						echo -e "\t\t\t<dependency>"
-						echo -e "\t\t\t\t<groupId>${group_id}</groupId>"
-						echo -e "\t\t\t\t<artifactId>${artifact_id}</artifactId>"
-						echo -e "\t\t\t\t<version>${version}</version>"
-						echo -e "\t\t\t</dependency>"
-					done >> "${pom_file_name}"
-			fi
-		done
+	for property in "${development_properties[@]}"
```

Comment:
@lucasmiranda0, please rename `property` to `dependency_property` to keep consistency